includes/site-health/load.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Site Health checks loader.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+if ( ! has_filter( 'user_has_cap', 'wp_maybe_grant_site_health_caps' ) ) {
```

Comment:
```suggestion
// Bail early if Site Health feature is disabled.
if ( ! has_filter( 'user_has_cap', 'wp_maybe_grant_site_health_caps' ) ) {
```

Reply:
Why are we even using this check and not actually `current_user_can( 'view_site_health_checks' )`?

Reply:
This is too early to check `current_user_can()`. It's not impossible, but not a good practice since the user is typically only initialized before `init`.

That said, I personally think we can remove this check entirely. It was present before to prevent users from activating useless modules, but that is no longer relevant. The filters that are added in the required files below aren't harmful even if Site Health is disabled. So I'd suggest to remove the check entirely.

Reply:
Removed in https://github.com/WordPress/performance/pull/1042/commits/88cd219b4b13d27f1045567bcb8d5d13091db1c9